ui/Screen/Onboarding/EnterName.svelte

```diff
@@ -68,9 +68,9 @@
       what should we call you?
     </h1>
     <p>
-      You’ll need a name to interact on Radicle. This isn’t unique across the
-      platform, but it helps others recognize you a little easier. You can
-      change it in your profile at any time.
+      You’ll need a display name to interact on Radicle. This isn’t unique
+      across the platform, but it helps others recognize you a little easier.
+      You can change it in your profile at any time.
```

Comment:
You can't change it yet in the B1.

```suggestion
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/892/commits/475ee1ea2246c55555e629fb29d0e1db87289a89